certified-connectors/DocuSignDemo/apiDefinition.swagger.json

```diff
@@ -286,8 +286,8 @@
         "tags": [
           "DocuSign"
         ],
-        "summary": "Get envelope docgen form fields",
-        "description": "Get the docgen fields from envelope documents.",
+        "summary": "Get document generation form fields from envelope",
+        "description": "Get document generation form fields from envelope",
```

Comment:
Looks like "Update DocGen" change did not happen.

Reply:
Done